netbox/extras/api/views.py

```diff
@@ -283,7 +283,7 @@ def run(self, request, pk):
 
         # Retrieve and run the Report. This will create a new Job.
         module, report_cls = self._get_report(pk)
-        report = report_cls()
+        report = report_cls
```

Comment:
The value is no longer a report *class*, so maybe clearer to drop report_cls so it becomes:

```
 module, report = self._get_report(pk)
```